habitat/omnitruck/hooks/init

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Removing previous version deployed at {{pkg.svc_static_path}}"
+rm -rf {{pkg.svc_static_path}}/* {{pkg.svc_static_path}}/.bundle
+
+echo "Deploying new version from {{pkg.path}} to {{pkg.svc_static_path}}"
+cp -a {{pkg.path}}/static/* {{pkg.path}}/static/.bundle {{pkg.svc_static_path}}
```

Comment:
Why do we need to copy the application code over to svc? Can't we just point the run hook to the pkg.path?